lib/season_statable.rb

```diff
@@ -6,39 +6,31 @@ def all_season_game_id(season)
     end.compact 
   end 
 
-  def winningest_coach(season)
+  def wins_per_coach(season) 
```

Comment:
definitely good to pull this out as it was done twice. Still a longer method, but I can't think of anything off the top of my head to make it shorter given the nested conditionals. Overall, this method is doing basically just one purpose which is good!